src/appleseed/renderer/kernel/shading/shadingpoint.cpp

```diff
@@ -411,10 +411,6 @@ void ShadingPoint::compute_world_space_partial_derivatives() const
             m_dpdu = (dv1 * dp0 - dv0 * dp1) * rcp_det;
             m_dpdv = (du0 * dp1 - du1 * dp0) * rcp_det;
 
-            // Make sure dPdv x dPdu points in the same direction as N.
```

Comment:
What's the rationale behind the removing of this code? Do these changes affect rendering of non-OSL bump mapping scenes?